src/main/java/duke/util/Storage.java

```diff
@@ -0,0 +1,106 @@
+package duke.util;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileNotFoundException;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+import java.text.ParseException;
+import java.util.ArrayList;
+import java.util.List;
+
+import duke.task.Deadlines;
+import duke.task.Events;
+import duke.task.Task;
+import duke.task.ToDos;
+
+/**
+ * Deals with loading tasks from the file and saving tasks in the file.
+ *
+ * @author marcuspeh
+ * @version A-JavaDoc
+ * @since 23 Aug 2021
+ */
+public class Storage {
+    /** File name for the storage. */
+    private static final String LOCATION_OF_FILE = "./data/duke.txt";
+    /** To deals with the errorMessages. */
+    private Ui ui;
+
+    /**
+     * Constructor for duke.main.Storage.
+     *
+     * @param ui to deal with the errorMessages.
+     */
+    public Storage(Ui ui) {
+        this.ui = ui;
+    }
+
+    /**
+     * Exports the task in list to a txt file in ./data.
+     *
+     * @param taskList list of task to be stored.
+     */
+    public void exportTask(List<Task> taskList) {
+        try {
+            File file = new File(LOCATION_OF_FILE);
+            file.getParentFile().mkdirs();
+            file.createNewFile();
+
+            FileWriter fileWrite = new FileWriter(file);
+            BufferedWriter bufferedWriter = new BufferedWriter(fileWrite);
```

Comment:
Really small issue, but I'm curious as to why BufferedWriter was initialised as bufferedWriter, but FileWriter fileWrite didn't follow this convention as well (fileWrite vs fileWriter).

Then again, since fileWrite isn't used anywhere else, you may also possibly consider instantiating it directly within bufferedWriter? Same with fileReader below, but this is probably entirely up to preference.

Reply:
Now that you pointed it out, I agree with what you mentioned about instantiating it directly. As for the naming convention, you raised a good point too.